frontend/modules/base/components/Footer.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import {
+    injectIntl,
+    FormattedMessage
+} from 'react-intl';
+
+class Footer extends React.Component{
+  render() {
+    return (
+      <footer className="footer">
+        <div className="container">
+          <p className="text-muted">
+            <FormattedMessage
```

Comment:
I've had to use the `FormattedMessage` tag instead of the usual `formattedMessage()` function because for some reason it won't let me use HTML in a placeholder.